brave/brave-core
# Adds support for Vimeo

Resolves https://github.com/brave/brave-browser/issues/4017
Resolves https://github.com/brave/brave-browser/issues/4391

UI PR: https://github.com/brave/brave-ui/pull/459

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- go to vimeo video
- click on the panel and make sure that you see publisher in the panel
- go to publisher page and make sure that publisher in the panel is correct
- start video and make sure that video is recorded in AC table

## Reviewer Checklist:
